doc/contributing/maintaining-postject.md

```diff
@@ -12,12 +12,12 @@ Check that Node.js still builds and tests.
 
 ## Committing postject
 
-1. Add postject: `git add --all test/fixtures/postject-copy`
-2. Commit the changes: `git commit`
+1. Add postject: `git add --all test/fixtures/postject-copy`.
+2. Commit the changes: `git commit`.
 3. Add a message like:
 
- ```text
- test: update postject to <version>
+```text
+test: update postject to <version>
 
- Updated as described in doc/contributing/maintaining-postject.md.
- ```
+Updated as described in doc/contributing/maintaining-postject.md.
+```
```

Comment:
This needs to be indented with three spaces so it's part of the list. You can play with https://astexplorer.net/ to convince you of this if that helps :)

Reply:
I had them in first place, but it's the command `make format-md` that deletes them :/

Reply:
I suppose that's because you used 1 space char instead of 3? When I try on my local install, `make format-md` keeps the indentation (as it should).

Reply:
I add a 3 space; let's see how it goes ^^

Just by curiosity, why 3?

Reply:
> Just by curiosity, why 3?

The long answer is https://spec.commonmark.org/0.30/#list-items. The short answer is "so it's aligned with the other list item content".

Reply:
I think that's because of this:

```
3. Add a message like:
^ ^
| |
+-+-- 3 characters
```